### PR TITLE
:seedling: set cma managed by addon-manager if not configured

### PR DESCRIPTION
**Pull request by haoqing0110**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/addon/controllers/managementaddon/controller.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	addonv1alpha1client "open-cluster-management.io/api/client/addon/clientset/versioned"
+	addoninformerv1alpha1 "open-cluster-management.io/api/client/addon/informers/externalversions/addon/v1alpha1"
+	addonlisterv1alpha1 "open-cluster-management.io/api/client/addon/listers/addon/v1alpha1"
+	"open-cluster-management.io/sdk-go/pkg/patcher"
+
+	"open-cluster-management.io/ocm/pkg/common/queue"
+)
+
+// clusterManagementAddonController reconciles cma on the hub.
+type clusterManagementAddonController struct {
```
[the remark this answers is not in the document]
Comment (haoqing0110) [reply pkg/addon/controllers/managementaddon/controller.go:21]: @qiujian16  will add more description, for the name, any suggestion?

[the remark this answers is not in the document]
Comment (qiujian16) [reply pkg/addon/controllers/managementaddon/controller.go:21]: If the purpose is to set annotation, maybe just CMAManagedByController

[the remark this answers is not in the document]
Comment (haoqing0110) [reply pkg/addon/controllers/managementaddon/controller.go]: Have modified the controller and added more description.

**Review by qiujian16 — COMMENTED** [review]
/approve

/assign @zhujian7

**Pull request merged**